tests/drivers/gpio/gpio_basic_api/src/test_gpio_port.c

```diff
@@ -359,6 +359,10 @@ static int check_pulls(void)
 
 	/* Disconnect output */
 	rc = gpio_pin_configure(dev, PIN_OUT, 0);
```

Comment:
Should we have a `#define GPIO_DISCONNECTED 0` ?

Reply:
That makes some sense to me.